src/ui/navigator/navigationBar/VPNBottomNavigationBar.qml

```diff
@@ -58,36 +58,39 @@ Rectangle {
         border.color: VPNTheme.theme.ink
     }
 
-    Flickable{
-        id: flickable
-        clip: true
-        anchors.fill: root
-        contentWidth: layout.implicitWidth
-        interactive: contentX > 0 || contentWidth > width
-
-        RowLayout {
-            height: flickable.height
-            width: flickable.width
-
-            RowLayout {
-                id: layout
-                objectName: "navigationLayout"
-                Layout.alignment: Qt.AlignHCenter
-                spacing: flickable.width * .2 // TODO something better here
-
-                Repeater {
-                    model: navButtons
-                    delegate: VPNBottomNavigationBarButton {
-                        objectName: navObjectName
-                        _screen: VPNNavigator[screen]
-                        _source: checked ? (_hasNotification ? sourceCheckedNotification : sourceChecked) : (_hasNotification ? sourceUncheckedNotification : sourceUnchecked)
-                        accessibleName: VPNl18n[navAccessibleName]
-                        ButtonGroup.group: navBarButtonGroup
-
-                        Component.onCompleted: {
-                            if(objectName === "navButton-messages") root.messagesNavButton = this
-                        }
-                    }
+
+    RowLayout {
+        id: layout
+        objectName: "navigationLayout"
+
+        anchors.fill: parent
+        anchors.topMargin: VPNTheme.theme.vSpacingSmall / 2
+        anchors.leftMargin: {
+            var minNumberOfIcons = 3
+            var paddingFactor = 8 //How much we decrease horizontal margins by for each new icon added (for non-tablets)
```

Comment:
Suggestion: What do you think about declaring properties for those? It might be a bit easier to hunt them down and adjust if needed.